qemu/tests/cfg/sgx_cpu.cfg

```diff
@@ -0,0 +1,36 @@
+- sgx_cpu:
+    type = sgx_cpu
+    only Linux
+    kill_vm = yes
+    login_timeout = 240
+    image_snapshot = yes
+    use_mem = no
+    mem_devs = "epc0 epc1"
+    backend_mem_epc0 = "memory-backend-epc"
+    size_mem_epc0 = 64M
+    prealloc_epc0 = on
+    backend_mem_epc1 = "memory-backend-epc"
+    size_mem_epc1 = 28M
+    prealloc_epc1 = on
+    vm_sgx_epc_devs = sgx0 sgx1
+    vm_sgx_epc_memdev_sgx0 = epc0
+    vm_sgx_epc_node_sgx0 = 0
+    vm_sgx_epc_memdev_sgx1 = epc1
+    vm_sgx_epc_node_sgx1 = 0
+    RHEL.9:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
+    RHEL.8:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+    cpuid_pkg = "cpuid"
+    sgx_features = "sgx1 sgx2 sgx_lc"
+    variants:
+        - sgx_cpu_default:
+            check_cpuid_sgx_cmd = "cpuid -1 |grep -i 'replace_string.*= true'"
```

Comment:
move it outside variants, for dup check_cpuid_sgx_cmd definition at both line28&34

Reply:
Thanks, move dup cmd outside variants